spec/abstractops.html

```diff
@@ -1041,6 +1041,8 @@ <h1>ISO 8601 grammar</h1>
           TimeZoneIANANameComponent `/` TimeZoneIANANameTail
 
       TimeZoneIANALegacyName :
+          &gt; An ASCII-case-insensitive match for one of the following:
+          `Etc/GMT` ASCIISign UnpaddedHour
```

Comment:
If the same nonterminal is going to include legacy names and Etc/GMT±\<h> names, I think it should be named "Special" rather than "Legacy".

Reply:
Re-reading [Timezone identifiers](https://data.iana.org/time-zones/theory.html#naming), Etc/GMT±<h> is actually also a _legacy name_, so I think this change makes it actually more correct.